ega-data-api-dataedge/src/main/java/eu/elixir/ega/ebi/dataedge/config/SwaggerConfig.java

```diff
@@ -17,12 +17,14 @@
 
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
 import springfox.documentation.builders.PathSelectors;
 import springfox.documentation.builders.RequestHandlerSelectors;
 import springfox.documentation.spi.DocumentationType;
 import springfox.documentation.spring.web.plugins.Docket;
 import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
+@Profile("debug")

```

Comment:
I personally use "swagger" for this case but I am totally ok with this. If we for example in a later PR add a profile for disabling caching we can use the same generic "debug" profile.

Reply:
I don't think a swagger endpoint should be visible to end user. I also use swagger, but for testing thus the `debug` profile.

Reply:
Oh, I did not explain myself clearly. I was referring about the name of the profile :D. Anyway, not important.

Reply:
ok, sorry I misunderstood.
we should fix the profiles in some later PR, because the are all over the place, maybe figure out what we can group together